### PR TITLE
Add SLT to Template Plugin: ROIAling-3

### PR DESCRIPTION
**Pull request by steve-y**
### Details:
 - *Create template plugin reference tests for target operation: ROIAlign-3*

### Tickets:
 - *79749*

### CODE REVIEW

File: docs/template_plugin/tests/functional/op_reference/roi_align.cpp [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        result << "_poolingMode=" << param.poolingMode;
+        if (param.testcaseName != "") {
+            result << "_=" << param.testcaseName;
+        }
+        return result.str();
+    }
+
+private:
+    static std::shared_ptr<Model> CreateFunction(const ROIAlignParams& params) {
+        const auto featureMap = std::make_shared<opset1::Parameter>(params.featureMap.type, params.featureMap.shape);
```
Comment (wilson-seok) [inline docs/template_plugin/tests/functional/op_reference/roi_align.cpp:76]: Can you make dynamic shape input for the parameter?

Comment (steve-y) [reply docs/template_plugin/tests/functional/op_reference/roi_align.cpp]: PartialShape is applied to input parameter. Thanks for your comment.

**Review by wilson-seok — APPROVED** [review]
LGTM

**Comment by steve-y** [comment]
@iefode @ilyachur could you review this ticket?

**Review by ilyachur — APPROVED** [review]

**Pull request merged**